Plans/iOS-Swift6_Base.xctestplan

```diff
@@ -14,9 +14,9 @@
   "testTargets" : [
     {
       "target" : {
-        "containerPath" : "container:iOS-Swift.xcodeproj",
-        "identifier" : "7B64386726A6C544000D0F65",
-        "name" : "iOS-Swift-UITests"
```

Comment:
I'm not sure how this even worked before. It seems to have been targeting the wrong test target 🤔